certbot-nginx/certbot_nginx/_internal/parser_obj.py

```diff
@@ -4,6 +4,7 @@
 import abc
```

Comment:
Dead code? Why not just removing it instead in this case?

Reply:
I think deleting dead code is a great idea in general, but the plan was that this would eventually replace the current nginx parser and we merged what we had so far to break up the work a bit. I think we could delete this code and throw it in a branch if we wanted, but I think we should definitely check in with Erica before doing that if we wanted to go that route.

My personal preference for this PR is to just leave it without the modifications and a `type: ignore` like I initially suggested, but we/I/you can check in with Erica if you prefer.

Reply:
I finally add a global `type: ignore` directive, since there will have not side-effect since this code is not used (on the contrary to `certbot.compat.os`).

Reply:
Ah, it appears I'm a little too late to be jumping in here. A global `type: ignore` sounds right for this PR. We should probably revisit if we plan to continue the rewrite work in a larger discussion, and make our decision to keep or branch this code based on that.

Reply:
I've added this to the discussion section of next week's meeting notes.